src/app/shared/services/github.provider.ts

```diff
@@ -0,0 +1,64 @@
+import { ɵa as BaseLoginProvider } from 'angularx-social-login';
```

Comment:
Garbage character?

Reply:
Can't do anything. The library exports it this way.